src/Nest/Ingest/Processors/PipelineProcessor.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Linq.Expressions;
+using Newtonsoft.Json;
+
+namespace Nest
+{
+	/// <summary> Executes another pipeline.</summary>
+	[JsonObject(MemberSerialization.OptIn)]
+	[JsonConverter(typeof(ProcessorJsonConverter<PipelineProcessor>))]
+	public interface IPipelineProcessor : IProcessor
+	{
+		//TODO this property clashes with the Name property on the IProcessor, need to rename base in master
```

Comment:
TODO?

Reply:
We can't fix this in `6.x` this needs to be changed in `master` updated the TODO to mention this. 👍